index.html

```diff
@@ -47,10 +50,7 @@ <h2>Choose your favorite Ingredient</h2>
         <label for="vanillaRadio">Vanilla</label><br /><br />
       </form>
 
-      <h3 id="label">---</h3>
-      <img id="cake-img" src="" width="300" height="300" />
-      <ul class="ingredients"></ul>
-      <a id="recipe-url" href="#"><button>press here for the recipe</button></a>
+      <div id="recipes"></div>
```

Comment:
MB `id="recipesDiv"` would be a better option, because I always like to have the element type in its id to make it more clear. But it's up to you to decide.